pages/common/choose.md

```diff
@@ -0,0 +1,36 @@
+# choose
+
+> A human-friendly and fast alternative to cut and (sometimes) awk.
+> More information: <https://github.com/theryangeary/choose>.
+
+- Print the 5th item from a line (zero indexed):
+
+`choose {{5}}`
+
+- Print the 0th, 3rd, and 5th item from a line, where items are separated by ':' instead of whitespace:
```

Comment:
I'm not sure about the way you use `Nth`. To me, the first element is at endex 0 and there is no 0th element. Any opinions on this @bl-ue, @CleanMachine1?

Reply:
Depending on the underlying programming language and convention @marchersimon, things can start at 0 rather than 1.

Reply:
Makes sense. But regardless of that, as @ SethFalco pointed out on Gitter (I have no idea how to copy the link to that thread), the element at the first position of a list is the first element, regardless of the indexing.